app/models/conference.rb

```diff
@@ -81,6 +82,21 @@ def user_registered? user
   end
 
   ##
+  # Delete all EventSchedules that are not in the hours range
+  # After the conference has been successfully updated
+  def delete_event_schedules
+    if self.start_hour_changed? || self.end_hour_changed?
+      event_schedules = EventSchedule.select do |e|
```

Comment:
@Ana06 `has_many thought` Sorry I didn't understand this. Are you trying to say instead of conference, program should have one-to-many relation with event_schedules model?